Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -38,7 +38,79 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (message.Length == 0)
+            {
+                return "";
+            }
+            // make a array so I can find the longest word. 
+
+            var arrayOfLines = message.Split(Environment.NewLine);
+
+            // Find length of longest word 
+            var lengthLongestWord = GetLongestWordLength(arrayOfLines);
+
+            // print table 
+
+            var table = new StringBuilder();
+            AddTopOrBottomLineToTable(padding, lengthLongestWord, table);
+            AddEmptyLinesToTable(padding, lengthLongestWord, table);
+            AddTextToTable(padding, arrayOfLines, lengthLongestWord, table);
+            AddEmptyLinesToTable(padding, lengthLongestWord, table);
+            AddTopOrBottomLineToTable(padding, lengthLongestWord, table);
+            return table.ToString();
+
+        }
+
+        private static StringBuilder AddTextToTable(int padding, string[] arrayOfLines, int lengthLongestWord, StringBuilder table)
+        {
+            foreach (var word in arrayOfLines)
+            {
+                table.Append("|");
+                table.Append(String.Empty.PadRight(padding, ' '));
+                table.Append(word);
+                table.Append(String.Empty.PadRight(lengthLongestWord - word.Length + padding, ' '));
+                table.Append("|");
+                table.Append(Environment.NewLine);
+
+            }
+
+            return table; 
+        }
+
+        private static int GetLongestWordLength(string[] arrayOfLines)
+        {
+            var lengthLongestWord = 0;
+            foreach (var word in arrayOfLines)
+            {
+                if (word.Length > lengthLongestWord)
+                {
+                    lengthLongestWord = word.Length;
+                }
+            }
+
+            return lengthLongestWord; 
+        }
+
+        private static StringBuilder AddEmptyLinesToTable(int padding, int lengthLongestWord, StringBuilder table)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change return type to 'void'; not a single caller uses the returned value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4929752)

Reply:
solved